src/Navigation/Navigate.ts

```diff
@@ -36,6 +36,9 @@ export const onNavigateAction = ({
   screenConditionConfigMap,
   setState,
 }: OnActionAttributes): boolean => {
+  if (action.type !== 'NAVIGATE') {
```

Comment:
why is this necessary?

Reply:
TypeScript shows an error that navigate specific attributes (`payload`, `flow`, `reset`, `skipConditionalNavigation`) are missing.

`return false` means that action was not handled, but this should never happen because the function is only called when `type === 'NAVIGATE'`, I just needed to tell TypeScript that.